### PR TITLE
fix(txbuilder): send assets after consuming requests

### PR DESCRIPTION
**Pull request by dessaya**

### CODE REVIEW

File: clients/iscmove/iscmoveclient/client_anchor_ptb.go [hunk 1]
```diff
@@ -225,12 +225,22 @@ func PTBTakeAndPlaceToAssetsBag(
...
+	Target iotago.Address
+	Assets iscmove.Assets
+}
+
 func PTBReceiveRequestsAndTransition(
 	ptb *iotago.ProgrammableTransactionBuilder,
 	packageID iotago.PackageID,
 	argAnchor iotago.Argument,
-	requestRefs []iotago.ObjectRef,
-	requestAssets []*iscmove.AssetsBagWithBalances,
```
Comment (lmoe) [inline clients/iscmove/iscmoveclient/client_anchor_ptb.go:233]: We had the issue that the isc.Assets don't hold 0 base token, which is why I introduced the inclusion of the AssetsBagWithBalance. I can't right now see if this reverts my change. Does it?

**Review by lmoe — APPROVED** [review]

File: clients/iscmove/iscmoveclient/client_anchor_ptb.go (hunk 1 above)
Comment (dessaya) [reply clients/iscmove/iscmoveclient/client_anchor_ptb.go:233]: The ConsumedRequest object still contains the AssetsBagWithBalances, so I don't think there is a change in that regard.

Comment (dessaya) [reply clients/iscmove/iscmoveclient/client_anchor_ptb.go:233]: But maybe I'm missing something. Was there a test for that edge case?

Comment (lmoe) [reply clients/iscmove/iscmoveclient/client_anchor_ptb.go:233]: Yes, a single one:

https://github.com/iotaledger/isc-rebased/pull/558/files#diff-12be2b5ad3e6179512722b78760906ac59fa8f20667fcc9761e8843487bf341cR167

Comment (dessaya) [reply clients/iscmove/iscmoveclient/client_anchor_ptb.go:233]: That test seems to be passing, so I guess this is fine
